### PR TITLE
DDF-5817 upgraded geotools to 20.1

### PR DESCRIPTION
**Pull request by stustison**
#### What does this PR do?
Upgrades Geotools

#### Who is reviewing it? 

@SmithJosh @bakejeyner 

#### Ask 2 committers to review/merge the PR and tag them here.
@blen-desta
@garrettfreibott
@pklinef
@rzwiefel
@shaundmorris
@stustison

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by stustison** [comment]
build now

File: catalog/measure/catalog-measure-api/src/main/java/ddf/measure/Distance.java [hunk 1]
```diff
@@ -53,19 +52,19 @@ private double getAsMeters(double distance, LinearUnit unitOfMeasure) {
 
     switch (unitOfMeasure) {
       case FOOT_U_S:
-        convertedDistance = NonSI.FOOT_SURVEY_US.getConverterTo(SI.METER).convert(distance);
```
Comment (bakejeyner) [inline catalog/measure/catalog-measure-api/src/main/java/ddf/measure/Distance.java]: USA USA USA !!!

⚾️ 🌭 🍔

File: distribution/test/test-metacard-validator/pom.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bakejeyner) [inline distribution/test/test-metacard-validator/pom.xml]: ❓ Was all this added code moved from somewhere else?

Comment (SmithJosh) [reply distribution/test/test-metacard-validator/pom.xml]: Looks like it came from here: https://github.com/codice/ddf-sdk

**Pull request merged**